packages/service-content/src/lib/resources/services/builders-locations.sync.ts

```diff
@@ -0,0 +1,73 @@
+import { z } from 'zod';
+
+import type { Dependencies } from '../../dependencies.js';
+import {
+  getBuildersWithoutLocationQuery,
+  setBuilderLocationQuery,
+} from '../queries/builders-locations.js';
+
+const expectedResponseSchema = z.array(
+  z
+    .object({
+      name: z.string(),
+      place_id: z.number(),
+      place_rank: z.number(),
+      display_name: z.string(),
+      lat: z.string(),
+      lon: z.string(),
+    })
+    .transform((data) => ({
+      placeId: data.place_id,
+      lat: Number.parseFloat(data.lat),
+      lng: Number.parseFloat(data.lon),
+    })),
+);
+
+const fetchBuilderLocation = async (query: string) => {
+  const q = encodeURIComponent(query);
+
+  const res = await fetch(
+    `https://nominatim.openstreetmap.org/search?format=jsonv2&limit=1&q=${q}`,
+  );
+  console.log('Buscando ubicación para:', query);
```

Comment:
Hello, english please ;)

Reply:
Hi Maxi, I’ve already fixed it. 
Thank you very much for the review ;)